extensions/typescript-language-features/src/utils/fileSchemes.ts

```diff
@@ -7,11 +7,13 @@ export const file = 'file';
 export const untitled = 'untitled';
 export const git = 'git';
 export const walkThroughSnippet = 'walkThroughSnippet';
+export const zip = 'zip';
 
 export const supportedSchemes = [
 	file,
 	untitled,
-	walkThroughSnippet
+	walkThroughSnippet,
+	zip
```

Comment:
This is temporary and will be replaced by the strategy described in the description, if greenlighted.